python/TestHarness/testers/Tester.py

```diff
@@ -41,6 +41,7 @@ def validParams():
 
         params.addParam('valgrind', 'NONE', "Set to (NONE, NORMAL, HEAVY) to determine which configurations where valgrind will run.")
         params.addParam('tags',      [], "A list of strings")
+        params.addParam('max_buffer_size', None, "Bytes allowed in stdout/stderr before it is subjected to being trimmed. Set to -1 to ignore output size restrictions")
```

Comment:
Doc comment says -1, you are defaulting to None (which isn't documented). I assume None is the old behavior of trimming? You should probably mention what the default is.